odoo/addons/base/tests/test_expression.py

```diff
@@ -1659,7 +1660,7 @@ def test_regular(self):
             ))
             ORDER BY "res_users"."id"
         ''']):
-            self.User.search([('groups_id.color', '=', group.color)], order='id')
+            self.User.search([('groups_id.color', '=', group_color)], order='id')
```

Comment:
It is mandatory, the ` group.color` make a additional request. Because the `group.name` doesn't fetch other fields like before (because it is translatable and not the rec_name -> not prefetchable)

Reply:
The problem is that it looks like cheating, like if an extra query was found in the main code, and you compensate by moving out some code.  In fact, `group.color` was free before, and it no longer is.